src/pages/cityWeather/CityWeather.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import { publicApiInstance } from '../../utils/api';
+import { WeatherCard, SearchInput, SearchBar, Tooltip } from '../../components';
+import endpoints from '../../config/enpoints';
+import { numberThree, numberZero, numberOne } from '../../config/constants';
+import * as S from './CityWeather.styles';
+
+const CityWeather = () => {
+  const [cityName, setCityName] = useState('');
+  const [selectedCity, setSelectedCity] = useState('');
+  const [results, setResults] = useState([]);
+  const [chooseCityForecast, setChooseCityForecast] = useState([]);
+
+  useEffect(() => {
+    if (cityName.length > numberThree) {
+      loadCities();
+    }
+  }, [cityName, loadCities]);
+
+  const cityValueHandler = value => {
+    setCityName(value);
+  };
+
+  const loadCities = async () => {
+    try {
+      const res = await publicApiInstance.get(endpoints.GET_CITIES(cityName));
+      setResults(res.data.locations);
+    } catch (error) {
+      alert(error);
+    }
+  };
+
+  const getCurrentCity = async city => {
+    try {
+      const res = await publicApiInstance.get(endpoints.GET_CITY_BY_ID(city.id));
+      setChooseCityForecast(res.data.forecast);
+      setCityName('');
+      setResults([]);
+      setSelectedCity(city.name);
+    } catch (error) {
+      alert(error);
+    }
+  };
+
+  return (
+    <div>
+      <S.Title>Find forecast for your favorite city!</S.Title>
+      <Tooltip
+        text="Enter at least 4 letters!"
+        tooltip={cityName.length >= numberOne && cityName.length <= numberThree}
```

Comment:
Number of letters in tooltip message can be dynamic and use the same variable you have in condition for cities loading
Condition `>=1` can be simplified to `>0` so you won’t need to use number 1

Reply:
Great! Thank you. I fixed 👍